TerminalDocs/.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
I'd either switch this to `true`, or add a second template for creating issues directly, not through the feedback control.

As you've got it now in this PR, visitors to your repo must use this template no matter what.

Reply:
Thanks Bill!